### PR TITLE
fix format shown date and change mock value

### PR DESCRIPTION
**Pull request by maxkanin**
## **Ticket**

**JIRA ID:** [NP-70](https://parking-system.atlassian.net/browse/NP-70)

## Changes proposed in this pull request
- [IMPORTANT] Reviewer **will not review** any PR without summary of changes
-fix format for shown date
-change mock value
please fill all the required fields from below Checklist otherwise reviewer **will not review** this PR
## Checklist
- [x] Screenshot evidence for both IOS and Android
- [x] Total change is less than 500 lines of code

## **Screenshot Evidence**
image
image
image
image

[NP-70]: https://parking-system.atlassian.net/browse/NP-70?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: utils/date.ts [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
   return getFormatedDate(date, "YYYY-MM-DD");
 };
-
+export const formatHumanReadableDateFromDateString = (date?: string | null) => {
+  if (date) {
+    const dateObject = parseISO(date);
+    return format(dateObject, "d MMMM yyyy");
+  }
+  return null;
+};
```
Comment (ingthanhunt) [inline utils/date.ts]: What is the case that date params will be null or undefined?

Comment (maxkanin) [reply utils/date.ts]: ตอนยังไม่ได้เลือกวันอะเพื่อที่ว่าจะได้โชว์ placeholderแทน

Comment (ingthanhunt) [reply utils/date.ts]: กูคิดว่าถ้าใช้ในกรณีเราควรไปกำหนด condition ตอนแรียกใช้แทนไหมนะ

Comment (maxkanin) [reply utils/date.ts]: โอเค

**Comment by maxkanin** [comment]
image-
image
image

**Review by ingthanhunt — APPROVED** [review]
lgtm

**Pull request merged**